app/helpers/form_helper.rb

```diff
@@ -315,7 +315,7 @@ def add_label options, f, attr
     required_mark = check_required(options, f, attr)
     label = ''.html_safe + options.delete(:label)
     label += ((clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name) &&
-      s_(clazz.gettext_translation_for_attribute_name attr).html_safe) if f && label.empty?
+      s_(clazz.gettext_translation_for_attribute_name attr).html_safe) if f.object && label.empty?
```

Comment:
If f is nil then `f.object` will throw `NoMethodError: undefined method 'owner' for nil:NilClass`.

I know this isn't your code but if you are already changing this maybe add the `respond_to?` part to the if? it's part of the condition before trying to set the label anyway.
Also, the if checks if the label is empty so there is no need to concatenate to the label.
Maybe something like:
```
 if f.try(:object) && label.empty? && (clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name)
    label = s_(clazz.gettext_translation_for_attribute_name attr).html_safe
 end
```